src/components/List.js

```diff
@@ -79,6 +80,21 @@ const List = ({ token }) => {
     return daysUntilNextPurchase > 0 ? daysUntilNextPurchase : 0;
   };
 
+  // Confirm that the user would like to delete this item
+  const confirmDelete = (item) => {
+    const confirmed = window.confirm(
+      'Are you sure you want to delete this item?',
```

Comment:
As long as you're passing in the item, it would be easy to add that to your message to make it more specific.
```suggestion
      `Are you sure you want to delete ${item.itemName}?`,
```